suites/quincy/cephadm/tier-0.yaml

```diff
@@ -16,6 +16,8 @@ tests:
       desc: Install software pre-requisites for cluster deployment.
       module: install_prereq.py
       name: setup pre-requisites
+      config:
```

Comment:
@manasagowri i think you forgot to remove that config from here as well, before adding file to git stage just cross check the files to be commited.

Reply:
Yes sunil thanks again. I have updated it.